common/hyperscaler/rules/rule.go

```diff
@@ -0,0 +1,225 @@
+package rules
+
+import (
+	"fmt"
+	"sort"
+)
+
+type Labels struct {
+	Labels []string
+}
+
+type Rule struct {
+	Plan                           string
+	PlatformRegion                 string
+	HyperscalerRegion              string
+	EuAccess                       bool
+	Shared                         bool
+	Labels                         map[string]string
+	ContainsInputAttributes        bool
+	ContainsOutputAttributes       bool
+	hyperscalerNameMappingFunction func(string) string
+}
+
+func NewRule() *Rule {
+	return &Rule{
+		hyperscalerNameMappingFunction: getHyperscalerName,
+		Labels:                         make(map[string]string),
+	}
+}
+
+type MatchableAttributes struct {
```

Comment:
Am I correct that this is structure representing data which will be given and used to generate correct labels.
We have MatchableAttributes and rules, and eventually we should get proper labels.

Reply:
No, labels are applied in attributes from definitions.go